apps/web/src/views/Activity/Activity.tsx

```diff
@@ -36,37 +44,50 @@ export const Activity = () => {
   };
 
   return (
-    <Flex flexDirection="column" height="full">
-      <Center display={isLoading && isFirstLoad ? "flex" : "none"} height="100%">
-        <Image width="150px" height="75px" marginBottom="136px" src={loadingWheel} />
-      </Center>
+    <>
+      <Flex zIndex={1} flexDirection="column" flexGrow={1}>
+        <Center display={isLoading && isFirstLoad ? "flex" : "none"} height="100%">
+          <Image width="150px" height="75px" marginBottom="136px" src={loadingWheel} />
+        </Center>
 
-      {operations.length === 0 && !isLoading && (
-        <EmptyMessage subtitle="activity" title="Activity" />
-      )}
-      {operations.length > 0 && (
-        <Box borderRadius="8px" onScroll={onScroll}>
-          {operations.map((operation, i) => {
-            const isFirst = i === 0;
-            const isLast = i === operations.length - 1;
+        {isVerified ? (
+          isEmpty && (
+            <EmptyMessage
+              margin="auto"
+              cta="Buy Tez Now"
+              ctaUrl={buyTezUrl}
+              subtitle={"You need Tez to take part in any activity.\n Buy some to get started."}
+              title="Welcome to Your Web3 Wallet"
+            />
+          )
+        ) : (
+          <VerifyMessage />
+        )}
+        {operations.length > 0 && (
+          <Box borderRadius="8px" onScroll={onScroll}>
+            {operations.map((operation, i) => {
+              const isFirst = i === 0;
```

Comment:
IIRC there are `_first` and `_last` props available for chakra components

Reply:
this code isn't in related to this PR
either way _last won't work since scroll loader is hidden and it's considered the last item 
<img width="1119" alt="image" src="https://github.com/user-attachments/assets/0df9cd56-0306-40b2-a83d-b921be363887">